src/io/dicom.ts

```diff
@@ -81,12 +84,12 @@ export class DICOMIO {
     await this.initialize();
 
     const inputs = await Promise.all(
-      files.map(async (file) => {
+      files.map(async (file, index) => {
         const buffer = await file.arrayBuffer();
         return {
           type: InterfaceTypes.BinaryFile,
           data: {
-            path: file.name,
+            path: index.toString(10), // make each file name unique
```

Comment:
Seems unnecessary to specify the radix when the default is decimal anyways.

Reply:
Removed the radix.